scripts/vim-patch.sh

```diff
@@ -92,8 +92,7 @@ get_vim_sources() {
 }
 
 commit_message() {
-  printf 'vim-patch:%s\n\n%s\n\n%s' "${vim_version}" \
-    "${vim_message}" "${vim_commit_url}"
+  printf '%s\n\n%s' "${vim_message}" "${vim_commit_url}"
```

Comment:
Will runtime patches still have the `vim-patch:hash` token ?

Reply:
Fixed.